src/core/autotracker.h

```diff
@@ -83,7 +83,7 @@ class AutoTracker final : public LuaInterface<AutoTracker>{
                 _snes->setMapping(USB2SNES::Mapping::SA1);
             _snesMapping = _snes->getMapping();
         }
-        if (strcasecmp(platform.c_str(), "n64") == 0) {
+        if (strcasecmp(platform.c_str(), "n64") == 0 or flags.find("lua") != flags.end()) {
```

Comment:
`or` in C++ is `||`
I would prefer `luaconnector` as the flag since `lua` could mean a lot of things - it's very ambiguous.
```suggestion
        if (strcasecmp(platform.c_str(), "n64") == 0 || flags.find("luaconnector") != flags.end()) {
```